lsp/roslyn.lua

```diff
@@ -0,0 +1,124 @@
+local sysname = vim.uv.os_uname().sysname:lower()
+local iswin = not not (sysname:find("windows") or sysname:find("mingw"))
+
+-- TODO(seb): Remove this in a couple of months after release
+local function try_resolve_legacy_path()
+    local legacy_path = vim.fs.joinpath(vim.fn.stdpath("data"), "roslyn", "Microsoft.CodeAnalysis.LanguageServer.dll")
+
+    if vim.uv.fs_stat(legacy_path) then
+        vim.notify(
+            "The default cmd location of roslyn is deprecated.\nEither download through mason, or specify the location through `vim.lsp.config.roslyn.cmd` as specified in the README",
```

Comment:
I'm getting this warning (twice) when opening files.

My config looks like this:

```lua
vim.lsp.config['roslyn'] = {
  cmd = {
    "dotnet",
    vim.fs.joinpath(vim.fn.stdpath("data"), "roslyn", "Microsoft.CodeAnalysis.LanguageServer.dll"),
    "--logLevel=Information",
    "--extensionLogDirectory=" .. vim.fs.dirname(vim.lsp.get_log_path()),
    "--stdio",
  },
  filetypes = { 'cs' },
}
```
I'm still using the _legacy_ path, and would like to keep doing so. Shouldn't this check that `vim.lsp.config.roslyn.cmd` is not defined instead?

Reply:
Good point, I will fix